cli/tests/integration_tests.rs

```diff
@@ -2200,6 +2200,81 @@ async fn inspector_break_on_first_line() {
   child.kill().unwrap();
 }
 
+#[tokio::test]
+async fn inspector_coverage() {
+  let script = deno::test_util::root_path()
+    .join("cli")
+    .join("tests")
+    .join("inspector_coverage.js");
+  let mut child = util::deno_cmd()
+    .arg("run")
+    // Warning: each inspector test should be on its own port to avoid
+    // conflicting with another inspector test.
+    .arg("--inspect=127.0.0.1:9232")
```

Comment:
@piscisaureus I tried changing port to multiple values, but when running all tests I'm getting either:
```
thread 'inspector_coverage' panicked at 'Can't connect: Io(Custom { kind: Other, error: "" })', cli/tests/integration_tests.rs:2224:28
```
or:
```
thread 'inspector_coverage' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Os { code: 32, kind: BrokenPipe, message: "Broken pipe" })', cli/tests/integration_tests.rs:2273:20
```
Any ideas? Doesn't matter which port I use

Reply:
Only happens when running tests on multiple threads - can't repeat when using `--test-threads=1`